Etabs_Adapter/CRUD/Create/Level.cs

```diff
@@ -55,14 +55,33 @@ private bool CreateCollection(IEnumerable<Level> levels)
                 Engine.Reflection.Compute.RecordWarning("Unnamed levels have been given name according to their height index: Level 'i'");
 
             string[] names = levelList.Select((x,i) => string.IsNullOrWhiteSpace(x.Name) ? "Level " + i.ToString() : x.Name).ToArray();
+            double[] heights = new double[count];   //Heights empty, set by elevations
+
+#if Debug16 || Release16
             double[] elevations = new double[count + 1];
 
             for (int i = 0; i < count; i++)
             {
                 elevations[i + 1] = levelList[i].Elevation;
             }
 
-            double[] heights = new double[count];   //Heights empty, set by elevations
+            if (elevations.Any(x => x <= 0))
```

Comment:
This check will always be true, as 0 level is inserted to the elevation (by not being assigned) and from memory, I think this is required for the API to play ball. Another check will be needed for etabs 2016.

Reply:
Right, should probably mention the difficulties in testing due to not having ETABS16